accelerator/errors.h

```diff
@@ -158,6 +158,8 @@ typedef enum {
   /**< fail to init lock */
   SC_CONF_LOCK_DESTROY = 0x05 | SC_MODULE_CONF | SC_SEVERITY_FATAL,
   /**< fail to destroy lock */
+  SC_CONF_PARSER_ERROR = 0x06 | SC_MODULE_CONF | SC_SEVERITY_FATAL,
+  /**< Error in initializing yaml parser */
```

Comment:
Let's stick to the previous description format: `fail tio initialize yaml parser`